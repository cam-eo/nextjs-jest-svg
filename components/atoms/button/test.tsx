import { Button } from ".";
import { render } from "@testing-library/react";

describe("Button", () => {
  it("should render Button", () => {
    const { asFragment } = render(<Button />);

    expect(asFragment()).toMatchSnapshot();
  });
});
