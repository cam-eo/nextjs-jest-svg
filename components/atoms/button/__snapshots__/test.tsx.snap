// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render Button 1`] = `
<DocumentFragment>
  <button />
</DocumentFragment>
`;
